polarify/main.py

```diff
@@ -36,41 +36,43 @@ def __init__(self, assignments: dict[str, ast.expr]):
         self.assignments = assignments
 
     @classmethod
-    def inline_expr(cls, expr: ast.expr, assignments: dict[str, ast.expr]) -> ast.expr:
-        return cls(assignments).visit(deepcopy(expr))
+    def inline_expr(cls, expr: ast.expr, assignments: dict[str, ast.expr]) -> ast.expr: # TODO
+        expr = cls(assignments).visit(deepcopy(expr))
+        assert isinstance(expr, ast.expr)
```

Comment:
Why is this required?

Reply:
Just a sanity check that it returns the right thing